### ISSUE TITLE
🐛 delta command gives output "You must specify a test script."

### ISSUE THREAD

**Issue by mstrYoda** [dandavison/delta#287]
I've install delta via "brew install delta" and when I run "delta" command it gives this output:

`You must specify a test script.`

Any idea why I can't run delta correctly?

**Comment by dandavison** [comment]
Hi @mstrYoda, the homebrew package is actually named `git-delta`. It's easy to get confused though! You'll have to `brew uninstall delta` and then `brew install git-delta`.

**Comment by mstrYoda** [comment]
Thank you so much, thats my bad.

**Comment by GsxCasper** [comment]
I have the same error message and I have both "git-delta" and "delta" brew packages installed (that's because I actually need the other "delta" package for other things). Is there a way I can make this work with both packages installed? @dandavison

**Comment by dandavison** [comment]
Hi @GsxCasper, I'm sorry about that. On the plus side, Rust executables are statically linked and in my experience are very easy to work with. So you could either download the MacOS executable from the [releases](https://github.com/dandavison/delta/releases) page, or even just [build delta from source](https://github.com/dandavison/delta#build-delta-from-source). Once you've [installed rust](https://www.rust-lang.org/learn/get-started), that is simply a case of issuing `cargo build --release` in the git repo, either on master or on the git tag for the latest release. You probably have your own scheme for this sort of thing but personally, what I do is have a directory `~/bin` which is on my shell `$PATH`, and then place symlinks in that directory to whatever executables I want to use (so in this case, to `target/release/delta` in the delta repo.)

**Comment by geofflangenderfer** [comment]
can we add these instructions to the readme?
```
brew install git-delta
```

**Comment by dandavison** [comment]
Hi @geofflangenderfer, the README links to https://dandavison.github.io/delta/installation.html, and that has the instructions for installation. As you can see from that, the installation instructions are quite bulky because people use package managers other than homebrew. And I was thinking of keeping the README clean and simple rather than having that entire table in it, every line but one of which is irrelevant to everybody.

**Comment by dandavison** [comment]
I've changed the beginning of the README so that it now reads as below. Hopefully that helps avoid this problem.

-----------------------------------------------------------

## Get Started

[Install it](https://dandavison.github.io/delta/installation.html) (the package is called "git-delta" in most package managers, but the executable is just `delta`) and add this to your `~/.gitconfig`:

**Comment by geofflangenderfer** [comment]
Dan, you're the man :)

thanks for your work!

**Comment by tgross35** [comment]
Could I maybe suggest to adjust the message to make this more clear? It's a tough problem to narrow down if you don't know what you're looking for, especially since it shows up in commands like `git branch -v`.

Something like "You must specify a test script." to "[delta]: You must specify a test script. 
 If you are attempting to use `delta` in Git, pease make sure you have `git-delta` installed instead of standard `delta`", just to make it a bit easier to find the solution

**Comment by dandavison** [comment]
@tgross35  sorry, the above discussion doesn't say it explicitly, but the problem is that the "test script" language comes from some completely unrelated (and little used) executable that is also named "delta". In several package managers (including homebrew) the other executable is what you get if you install a package named "delta" -- for my delta, you have to install a package named "git-delta".

**Comment by mazunki** [comment]
@dandavison: I think their point was that seeing as this is a "fairly common" mistake, the error message could hint at this issue instead of only saying "missing test script". That PR would need to go to that other program, and not here, though.

**Comment by dandavison** [comment]
Right, I'd obviously be happy for the other program to add clarification. I should make it clear though that the naming clash is my doing: the other program was around for years before my delta; I wasn't aware of it when I started delta, and decided that I didn't want to change the name when I found out about it. So it's possible that they would take a dim view of such a request!

(This situation has prevented delta from being carried by some package distribution systems and has even made at least one person angry! My personal opinion is that it's a bit hopeless to expect all human software created in all cultures throughout history to find unique slots in a single namespace, but admittedly that's not a terribly practical viewpoint and also admittedly I don't deal with the day-to-day challenges of running distros and package managers but am very grateful to the people that do).
